3.4.1/scala2.12-java11-ubuntu/Dockerfile

```diff
@@ -43,8 +43,8 @@ ENV SPARK_TGZ_URL=https://archive.apache.org/dist/spark/spark-3.4.1/spark-3.4.1-
 RUN set -ex; \
     export SPARK_TMP="$(mktemp -d)"; \
     cd $SPARK_TMP; \
-    wget -nv -O spark.tgz "$SPARK_TGZ_URL"; \
-    wget -nv -O spark.tgz.asc "$SPARK_TGZ_ASC_URL"; \
+    wget -nv -O spark.tgz "${SPARK_TGZ_URL}"; \
+    wget -nv -O spark.tgz.asc "${SPARK_TGZ_ASC_URL}"; \
```

Comment:
Is it a required change? otherwise I think we might better to recover.

Reply:
So while not technically required, this is a result of the changes in the PR to the Dockerfile.template following running the add-docker files.sh script against 3.4.1. I will back out the brackets in the template and rerun to remove these brackets from the generated Dockerfile here. Just documenting the cause of this change.

Reply:
Please note that since I merged in the changes from master in 81c2933 that had merged after this PR was initiated, that pulled in changes also to the entrypoint.sh.template. So after making the change to the Dockerfile.template and rerunning add-dockerfiles.sh, both the Dockerfile and entrypoint.sh for 3.4.1/2.12 has been changed.